sysfs.c

```diff
@@ -1269,3 +1269,33 @@ bool sysfs_is_libata_allow_tpm_enabled(const int verbose)
 		return true;
 	return false;
 }
+
+bool init_md_mod_param(char *buffer, int bsize)
```

Comment:
please add some function desc. I see that `*buffer` is used to get current state of legacy_async_del_gendisk but it is not obvious. This is side logic I would have described.